docs/deployment/code/apify/crawler_as_actor_example.py

```diff
@@ -1,4 +1,4 @@
-from apify import Actor
+from apify import Actor  # type: ignore[import-not-found]
```

Comment:
We should rather deal with this by adjusting the mypy settings in the `pyproject.toml`

Reply:
Yup, adding comments like this to our examples is a no-go.

Reply:
Done